authority/root.go

```diff
@@ -17,13 +17,39 @@ func (a *Authority) Root(sum string) (*x509.Certificate, error) {
 
 	crt, ok := val.(*x509.Certificate)
 	if !ok {
-		return nil, &apiError{errors.Errorf("stored value is not a *cryto/x509.Certificate"),
+		return nil, &apiError{errors.Errorf("stored value is not a *x509.Certificate"),
 			http.StatusInternalServerError, context{}}
 	}
 	return crt, nil
 }
 
 // GetRootCertificate returns the server root certificate.
 func (a *Authority) GetRootCertificate() *x509.Certificate {
-	return a.rootX509Crt
+	return a.rootX509Certs[0]
+}
+
+// GetRootCertificates returns the server root certificates.
```

Comment:
Here we have two methods in the public Authority API that return the same thing. The explanation behind this duplication is that one of these methods is used by the CA API while the other is used by the SDK API. It is possible that the implementations for the methods change (e.g. new conditions are added to one and not the other) in the future.

Reply:
Fixed in e8ac3f4